src/InputContainer.test.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fireEvent, render } from '@testing-library/react';
+
+import InputContainer from './InputContainer';
+
+jest.mock('react-redux');
+
+describe('InputContainer', () => {
+  const dispatch = jest.fn();
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    useDispatch.mockImplementation(() => dispatch);
+
+    useSelector.mockImplementation((selector) => selector({
+      taskTitle: '할 일1',
+    }));
+  });
+
+  it('입력된 값과 element들을 출력합니다.', () => {
+    const { queryByText, getByLabelText, getByPlaceholderText } = render((<InputContainer />));
```

Comment:
```suggestion
    const { queryByText, getByLabelText, getByPlaceholderText } = render(<InputContainer />);
```

한 줄일 때는 괄호가 하나면 충분해요 ;)

Reply:
여러 줄일때는 괄호를 두번 하는 이유가 무엇인지 모르겠습니다.
stackoverflow에 render double parentheses라고 검색했는데 적절한 답변을 못찾겠습니다. 어떤 부분을 다시 봐야할까요?